packages/data-point/lib/reducer-types/reducer-helpers/reducer-constant/resolve.js

```diff
@@ -0,0 +1,17 @@
+const Promise = require('bluebird')
```

Comment:
does this mean constants also need to get resolved?

Reply:
Constants inside of ReducerObjects do not use this function, but if you use a constant anywhere else it will use this resolve function.

Reply:
we probably don't need to wrap this return value in `Promise.resolve`, but I want to match the return type for other resolve functions. I think `constant` will mainly be used in reducer objects though, so I don't expect this function to be used very often.